src/content/en/fundamentals/performance/rail.md

```diff
@@ -78,18 +83,24 @@ For actions that take longer than 500ms to complete, always provide feedback.
 
```

Comment:
On line 72:
You have 100ms to respond to user input before they notice a lag.

This is a bit confusing, because it isn't true for touch drag, or scroll for example.